### ISSUE TITLE
Attribute error

### ISSUE THREAD

**Issue by dmolina213** [deralexxx/PySight2MISP#24]
I am getting this attribute error when i run PySight.py. Checked PySight_settings and  the attribute appears to be there.
-why is it looking for that value if i have both misp proxy and isight proxy set to false?
Traceback (most recent call last):
  File "PySight.py", line 873, in <module>
    result = isight_search_indicators(PySight_settings.isight_url, PySight_settings.isight_pub_key,PySight_settings.isight_priv_key, PySight_settings.isight_last_hours)
  File "PySight.py", line 746, in isight_search_indicators
    return isight_prepare_data_request(base_url, search_query, public_key, private_key)
  File "PySight.py", line 724, in isight_prepare_data_request
    result = isight_load_data(a_url, a_query, header)
  File "PySight.py", line 625, in isight_load_data
    PySight_settings.proxy_address)
A**ttributeError: module 'PySight_settings' has no attribute 'proxy_address'**

**Comment by dmolina213** [comment]
I think i got it working. however at the end of the process i either get it to core dump or get a messaged 'killed'. There was too much happening on my screen and my eyeball wasnt fast enough to catch anything significant.  Any ideas?

**Comment by dmolina213** [comment]
I was able to see it pulling down the events. I set the log mode to DEBUG and i did a tail -f output.log. Looks like it stopped here: 
2020-04-29 02:42:19,454 - PySight_settings - DEBUG - Processing report 20-00007633
DEBUG [api.py:2242 - _prepare_request() ] {'User-Agent': 'PyMISP 2.4.123 - Python 3.6', 'Accept-Encoding': 'gzip, deflate', 'Accept': 'application/json', 'Connection': 'keep-alive', 'Authorization': 'RCA3rzX3q8tozY3NTw2yzwU96NnFnWscEnOsR8HD', 'content-type': 'application/json'}
DEBUG:PySight_settings:Processing report 20-00007626
***threading****
Killed

However, when i looked at the events in the GUI, I could not find any of the indicators.

**Comment by jaegeral** [comment]
The error message meant that you have not set the proxy settings in the config file

If no events show up in the GUI, that means it did not run correctly.

**Comment by dmolina213** [comment]
I was able to get it to work. I had to disable threading on the script.

V/R
Help Me, Help You
Douglas P Molina, MBA, CISSP, C|EH, CySA+,CCSK,GCTI
U.S. Department of Veterans Affairs
Office of Information Security (OIS)
Cyber Threat Intelligence (CTI)
Work Phone: 708-938-2720
Mobile Phone: 202 779 2034
UNCLASS: douglas.molina@va.gov<mailto:douglas.molina@va.gov>
[cid:image001.jpg@01D621EB.389AAB60]
